### DISCUSSION TITLE
Can AirSim simulate ground effects?

### DISCUSSION THREAD

**Discussion by zhangbaozhe** [microsoft/AirSim#4039] [Support Q&A]
Hello there, 
Recently I‘ve been working on a project to simulate the ground effects (which can increase lift force and decrease drag force when a drone is near the surface).  So I'm wondering if AirSim can simulate these effects, or are there any air settings that I can change to let it do this kind of simulation (since I noticed that it was quite smooth when the drone taking off or landing in the default block scene).  
Thanks ;-)

**Comment by jonyMarino** [comment]
Hi @zhangbaozhe! AirSim does not take ground effects into account. If you want to modify that take a look at [FastPhysicsEngine](https://github.com/microsoft/AirSim/blob/master/AirLib/include/physics/FastPhysicsEngine.hpp)

**Comment by seanmcleod** [comment]
@jonyMarino how far did you get with the option to integrate and make use of JSBSim as the physics engine?

Here is an example of ground effects being modelled in JSBSim.

https://seanmcleod.github.io/2018/02/landing-ground-effect-flare-jsbsim/

**Comment by jonyMarino** [comment]
@seanmcleod Sorry for the late response. 
Yes. I forgot to say that an external connection with Gazebo or JSBSim can be used to have other physics. Thanks for referring to how to solve it with JSBSim. In principle, the integration through scripts made at https://github.com/AOS55/Fixedwing-Airsim can be used.
Regarding our work, I will answer you in the discussion that we have in the JSBSim repository.
